src/index.mjs

```diff
@@ -5,19 +5,27 @@ import * as fs from 'node:fs';
 import * as path from 'node:path';
 import process from 'node:process';
 
-import { cosmiconfigSync } from 'cosmiconfig';
+import confirm from '@inquirer/confirm';
+import { cosmiconfig } from 'cosmiconfig';
 import detectPackageManager from 'which-pm-runs';
 import { execa } from 'execa';
 import ora from 'ora';
 import picocolors from 'picocolors';
```

Comment:
[suggestion] From the POV of readability, `picocolors.` seems verbose to me.

How about shortening it like this?


```suggestion
import pc from 'picocolors';
```

or, to keep the diff minimum:


```suggestion
import picocolors from 'picocolors';
const pc = picocolors;
```

or, most shortened (personally preferred):


```suggestion
import { bold, red, green, ... } from 'picocolors';
```

Reply:
I went with `pc` as the package doesn't seem to have named exports at the moment.